src/components/SignInButtons/AppleSignIn/index.android.tsx

```diff
@@ -18,9 +18,9 @@ const config = {
 
 /**
  * Apple Sign In method for Android that returns authToken.
- * @returns {Promise<string>}
+ * @returns Promise that returns a string when resolved
  */
-function appleSignInRequest() {
+function appleSignInRequest(): Promise<string | undefined> {
```

Comment:
Do you think type `never` is more accurate?

https://stackoverflow.com/a/63372777

Reply:
@rushatgabhane 
![Screenshot 2024-01-04 at 16 16 10](https://github.com/Expensify/App/assets/46921547/b901528d-9d5e-4c5b-bad6-6e90c5200171)